Sources/Sentry/include/SentrySession.h

```diff
@@ -19,6 +19,7 @@ typedef NS_ENUM(NSUInteger, SentrySessionStatus) {
 SENTRY_NO_INIT
 
 - (instancetype)initWithReleaseName:(NSString *)releaseName;
+- (instancetype)initWithReleaseName:(NSString *)releaseName cacheDirectoryPath:(nullable NSString*)cacheDirectoryPath;
```

Comment:
`m`: We should only offer one initializer and remove the one with only one parameter. Only SentryHub calls this at the moment. As the hub has the options available we can pass in the `distinctID` here. So instead of the session calling `SentryInstallation` for the ID the SentryHub should do that and pass the distincID.